nucypher/blockchain/economics.py

```diff
@@ -56,6 +56,7 @@ class BaseEconomics:
 
     # Period Definition
     _default_hours_per_period = 24
+    _default_genesis_hours_per_period = 24
 
     # Time Constraints
     _default_minimum_worker_periods = 2
```

Comment:
For `_default_minimum_worker_periods = 2` and `_default_minimum_locked_periods = 30  # 720 Hours minimum`:

Would we want to adjust these relative to the changes made in hours per periods; for example, instead of a min locked periods of 30 periods (period is 24 hours in length), if it was decided the period length would become 48 hours, should this min locked periods be changed to 15 periods i.e. same min 720 hours? Or, are we ok with the implicit increase in default min values when increasing the hours per periods.

Reply:
Yes but this is not part of this PR, this touchs only code logic